encrypted-media-respec.html

```diff
@@ -2218,8 +2222,7 @@
 
         <section id="update-key-statuses">
           <h4>Update Key Statuses</h4>
-          <p>The Update Key Statuses algorithm is run when the CDM changes the set of keys <a href="#known-key">known</a> to the session or the status of one or more of the keys.
-          This can happen as the result of a <a def-id="load"></a> or <a def-id="update"></a> call or some other event, such as expiration.
+          <p>The Update Key Statuses algorithm updates the set of keys <a href="#known-key">known</a> to a <a>MediaKeySession</a> or the status of one or more of the keys.
```

Comment:
nit: "updates the set of keys known to" sounds a bit like this algorithm causes this to change when in reality it is updating the object to reflect the set of known keys. If we think of the `MediaKeySession` as a reflection of CDM state, then maybe that makes more sense.

Changing the wording a bit might help, though. For example, "...`updates the set of <a href="#known-key">known</a> keys for a <a>MediaKeySession</a>...`"